src/makielayout/defaultattributes.jl

```diff
@@ -11,8 +11,30 @@ function lift_parent_attribute(::Nothing, attr::Symbol, default_value)
 end
 
 
+function wong_colors(alpha)
+    colors = [
+        RGB(0/255, 114/255, 178/255), # blue
```

Comment:
I've also done some permutation in AlgebraOfGraphics (see [here](https://github.com/JuliaPlots/AlgebraOfGraphics.jl/blob/0d22e22e06a19d9bd90e4e3761fcc2ee38b57af7/src/theme.jl#L8)) so that the first few look nice together. I think it's the same, other than I have `vermillion` and `reddish purple` flipped, because it looked to me that vermillion and orange were a bit too similar and it was nice to delay having both together. (OTOH, I could take your names, I named them according to your old PR but these names are nicer.)